src/pages/Home.js

```diff
@@ -14,42 +14,36 @@ import Preview from '../components/Preview';
 export default function Home() {
   return (
     <>
-      <div className="wrapper">
```

Comment:
The `wrapper` here was just an extra element that was doing the same job as the parent element, so I removed it